### ISSUE TITLE
Trigger start a call from mattermost API

### ISSUE THREAD

**Issue by bayunvnt** [mattermost/mattermost-plugin-calls#309]
Hello, i tried to post a "/call start" from API "https://mattermost.domain.id/api/v4/commands/execute", but the calls is not started, there is any way to trigger call start from an api? thankyou

**Comment by streamer45** [comment]
@bayunvnt Nope, unfortunately it's not possible as a call only exists as long as at least one user is connected so to start a call you'd have to create a websocket connection and keep it alive. 

That said, in the near future we may be able to make this more automatable by relying on the recently introduced Calls bot. 

By the way, what you are looking to achieve exactly? Getting to understand your use case would be very helpful.

**Comment by bayunvnt** [comment]
thankyou for your response, i recently use mattermost for communicating with teams, everyday in the morning we need to do a meeting about taslisks, jobs and etc, i want to automate start a call in the morning with an energic message for the rest of teams so they can join directly without manually start a calls.

**Comment by streamer45** [comment]
Thanks for the context. I created an internal ticket ([MM-49965](https://mattermost.atlassian.net/browse/MM-49965)) to track this as it sounds like a very useful addition. Will give an update here as soon as get to implement it.
